packages/react/src/Prose/Prose.stories.tsx

```diff
@@ -63,7 +64,42 @@ const ExampleHtmlMarkup = `
     <p>Nunc velit odio, posuere eu felis eget, consectetur fermentum nisi. Aenean tempor odio id ornare ultrices. Quisque blandit condimentum tellus, semper efficitur sapien dapibus nec. </p>
 `
 
-export const Playground: ComponentStory<typeof Prose> = args => <Prose {...args} html={ExampleHtmlMarkup} />
+const UnorderedListHtmlMarkup = `
+<ul>
+  <li>
+    Vivamus eu risus nec lectus consequat rutrum at vel lacus.
+  </li>
+  <li>
+    Donec at dolor ut metus imperdiet congue vel porta nunc.
+  </li>
+  <li>
+    Quisque eu tortor suscipit, congue quam in, bibendum tellus.
+  </li>
+</ul>
+`
+
+export const Playground: StoryFn = args => <Prose {...args} html={ExampleHtmlMarkup} />
+const UnorderedListStory: StoryFn = args => <Prose {...args} html={UnorderedListHtmlMarkup} />
```

Comment:
Could we please move this story to `Prose.features.stories.tsx` so that it appears under `Features` in the nav? The only top-level stories we display should be `Default` and `Playground`. 

![Screenshot 2023-10-17 at 10 03 15](https://github.com/primer/brand/assets/13340707/85e9a665-c1dc-4141-b06e-e09d1cd8acf7)